tests/neo4j/txrun.py

```diff
@@ -29,9 +30,6 @@ def test_updates_last_bookmark_on_commit(self):
         self.assertGreater(len(bookmarks[0]), 3)
 
     def test_does_not_update_last_bookmark_on_rollback(self):
-        if get_driver_name() in ["java"]:
-            self.skipTest("Rollback not implemented in backend")
```

Comment:
I've implemented the rollback in the java driver testkit-backend to be able to test the other stuff, so it's working now.

Reply:
Actually, I did this much more time ago because of another test-case, but I've forgot to remove the skip.

https://github.com/neo4j/neo4j-java-driver/pull/887/files